### PR TITLE
【映射文档】新增与维护部分映射文档 part2

### PR DESCRIPTION
**Pull request by RedContritio**
- 新增了部分映射文档，基于 PaConvert 已实现的 api 映射

> 尚有约 50 个已实现的 api 映射尚未添加映射文档。

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.IndependentTransform.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
...
 ### 参数映射
 
-| PyTorch                   | PaddlePaddle             | 备注                                                 |
-| ------------------------- | ------------------------ | ---------------------------------------------------- |
-| base_transform            | base                     | 基础变换，仅参数名不一致。                           |
-| reinterpreted_batch_ndims | reinterpreted_batch_rank | 被扩展为事件维度的最右侧批维度数量，仅参数名不一致。 |
-| cache_size                | -                        | cache 大小，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。   |
+| PyTorch                   | PaddlePaddle             | 备注 |
+| ------------------------- | ------------------------ | -- |
+| base_transform            | base                     | 基础变换。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.IndependentTransform.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.IndependentTransform.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
...
+| reinterpreted_batch_ndims | reinterpreted_batch_rank | 被扩展为事件维度的最右侧批维度数量，需大于 0。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.IndependentTransform.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.celu.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+paddle.nn.functional.celu(x, alpha=1.0, name=None)
+```
+
+其中 PyTorch 和 Paddle 功能一致，仅参数名不一致，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注 |
+| ------- | ------------ | -- |
+| input   | x            | 输入的 Tensor。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.celu.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dropout.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+paddle.nn.functional.dropout(x, p=0.5, axis=None, training=True, mode='upscale_in_train', name=None)
+```
+
+其中 PyTorch 和 Paddle 功能一致，仅参数名不一致，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注 |
+| ------- | ------------ | -- |
+| input   | x            | 输入的多维 Tensor，数据类型为：float16、float32、float64。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dropout.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dropout.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+| p       | p            | 将输入节点置 0 的概率，即丢弃概率。默认值为 0.5。 |
+| train   | training     | 标记是否为训练阶段。默认值为 True。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dropout.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.eq.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+paddle.equal(x, y)
+```
+
+PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                         |
+| ------- | ------------ | ------------------------------------------------------------ |
+| input   | x            | 输入 Tensor，支持的数据类型包括 bool、float16、float32、float64、uint8、int8、int16、int32、int64。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.eq.md]: 仅参数名不一致

bool、float16这些无关信息就不需要了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.eq.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+| other   | y            | 输入 Tensor，支持的数据类型包括 bool、float16、float32、float64、uint8、int8、int16、int32、int64。                                           |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.eq.md]: 仅参数名不一致

bool、float16这些无关信息就不需要了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.ge.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+paddle.greater_equal(x, y)
+```
+
+PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                         |
+| ------- | ------------ | ------------------------------------------------------------ |
+| input   | x            | 输入 Tensor，支持的数据类型包括 bool、float16、float32、float64、uint8、int8、int16、int32、int64。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.ge.md]: 仅参数名不一致

bool、float16这些无关信息就不需要了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.ge.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+| other   | y            | 输入 Tensor，支持的数据类型包括 bool、float16、float32、float64、uint8、int8、int16、int32、int64。                                           |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.ge.md]: 仅参数名不一致

bool、float16这些无关信息就不需要了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool1d.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+## [ torch 参数更多 ]torch.max_pool1d
+
+### [torch.max\_pool1d](https://pytorch.org/docs/stable/jit_builtin_functions.html#supported-pytorch-functions)
+
+```python
+torch.max_pool1d(input: Tensor,
+                 kernel_size: List[int],
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool1d.md]: 类型标注 可以删掉，免得签名看起来太复杂

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool2d.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+## [ torch 参数更多 ]torch.max_pool2d
+
+### [torch.max\_pool2d](https://pytorch.org/docs/stable/jit_builtin_functions.html#supported-pytorch-functions)
+
+```python
+torch.max_pool2d(input: Tensor,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool2d.md]: 类型标注 可以删掉，免得签名看起来太复杂

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool3d.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
+## [ 仅参数名不一致 ]torch.max_pool3d
+
+### [torch.max\_pool3d](https://pytorch.org/docs/stable/jit_builtin_functions.html#supported-pytorch-functions)
+
+```python
+torch.max_pool3d(input: Tensor,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.max_pool3d.md]: 类型标注 可以删掉，免得签名看起来太复杂

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.mean.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+paddle.mean(x, axis=None, keepdim=False, name=None)
+```
+
+其中 PyTorch 和 Paddle 功能一致，仅参数名不一致，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注 |
+| ------- | ------------ | -- |
+| input   | x            | 输入 Tensor。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.mean.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.mean.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+| dim     | axis         | 指定对 x 进行计算的轴。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.mean.md]: 仅参数名不一致

**Review by zhwesky2010 — COMMENTED** [review]
注意格式规范
注意参数映射里不用过多介绍这个参数，重点是**突出两个参数的差异**，而不是这个参数的功能

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.AffineTransform.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch    | PaddlePaddle | 备注 |
+| ---------- | ------------ | -- |
+| loc        | loc          | 偏置参数。 |
+| scale      | scale        | 缩放参数。 |
+| event_dim  | -            | event_shape 可选尺寸。对于单随机变量为 0，对于向量分布为 1，对于矩阵分布为 2。Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.AffineTransform.md:23]: 这个可直接删除吗，没有影响吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft2.md [hunk 16]
```diff
@@ -1,35 +1,35 @@
...
-| ----------------------------------- | ------------ | ----------------------------------------------------------------------- |
+| PyTorch | PaddlePaddle | 备注  |
+| ------- | ------------ | ----- |
 | input     | x           | 表示输入的 Tensor ，仅参数名不一致。                         |
-| n     | n           | 表示在傅里叶变换轴的长度 。                         |
+| s         | s           | 表示在傅里叶变换轴的长度 。                         |
 | dim       | axis        | 表示进行运算的轴，仅参数名不一致。                           |
-| norm     | norm           | 表示傅里叶变换的缩放模式。                         |
-| out           | -      | 表示输出的 Tensor ， Paddle 无此参数，需要转写。         |
+| norm      | norm        | 表示傅里叶变换的缩放模式。                         |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft2.md:24]: torch的None和 paddle的`'backward'` 是相同效果的吗

**Review by zhwesky2010 — APPROVED** [review]
下个PR看下这两个问题

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.AffineTransform.md (hunk 15 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.AffineTransform.md:23]: paddle 没有描述具体实现，可能需要实验来测试。
认为没有影响是因为参考 [PaConvert/#341](https://github.com/PaddlePaddle/PaConvert/pull/341)，对该参数做了直接删除处理。
此外 `tests/test_distributions_AffineTransform.py` 未覆盖 0、1、2 取值。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft2.md (hunk 16 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft2.md:24]: 这里 torch 会在该参数未指定或传入 None 的时候，内部使用默认值 "backward" （不依赖 python 语法的手动默认值设置）。

效果一致。
